doc/articles/contributing/guidelines/creating-a-new-release.md

```diff
@@ -0,0 +1,13 @@
+# Guidelines for releasing Uno
+
+## Overview
+
+When a pull-request is merged to master a new development version is automatically pushed to NuGet.org. A development version can be identified by the `-dev` prefix ie. `major.minor.patch-dev.2294`.
```

Comment:
It is more like suffixed with `-dev`.
the `-dev` suffix; or
the `-dev` tag?

Reply:
nice catch. Yes, it's a suffix, not prefix.